themes/gatsby-theme-catalyst-bery/src/components/header.js

```diff
@@ -161,10 +161,11 @@ const SiteHeader = () => {
         </div>
         <nav
           sx={{
-            mt: [3, null, 4, null, null],
+            mt: [4, null, null, null, null],
             display: "flex",
             gridColumn: "1 / -1",
             gridRow: "4 / 5",
+            placeContent: ["center", "start", null, null, null],
```

Comment:
Can you say more about your thinking with the center alignment? I prefer the left alignment as it is consistent with the avatar image and also consistent with the text below it...but I can also see benefits to the center align as well.

<img width="423" alt="Screen Shot 2020-08-15 at 10 38 44 PM" src="https://user-images.githubusercontent.com/204841/90326648-17504f80-df48-11ea-84cc-8f6765bfbd61.png">